### ISSUE TITLE
Modern mode build option

### ISSUE THREAD

**Issue by richterdennis** [quasarframework/quasar#5899]
@VitalyEmelyanov requested [here](https://github.com/quasarframework/quasar-cli/issues/169) a modern mode build option. This is still relevant to reduce build size and improve performance in modern browsers (old browsers are still supported).

> Vue CLI 3 has a new feature for build - [modern mode](https://cli.vuejs.org/guide/browser-compatibility.html#modern-mode). It creates 2 versions of scripts for modern and old browsers, linked in specific way through `<script type="module">`.
I'd like to see such option in quasar-cli, because it can reduce build size and improve performance in modern browsers greatly.

For newer browsers the transpiled ES5 code bundled with polyfills is unnecessary and slower then ES6+ code. Please add an option to quasar.conf, which will then if enabled create two bundles, one modern es6+ bundle and one legacy es5 bundle.

**Comment by rstoenescu** [comment]
It's coming. Stay tuned.

**Comment by rstoenescu** [comment]
Modern mode will be available in "@quasar/app" v1.9.
```bash
$ quasar build [....] --modern
$ quasar dev [...] --modern

# or through quasar.conf.js > build > modern: true
```
